src/esp_loader.ts

```diff
@@ -768,7 +767,9 @@ export class ESPLoader extends EventTarget {
   getBootloaderOffset() {
```

Comment:
We have so many chip family checks in supported methods like mac address or bootloader offset, that it feels like we should consider adding this extra info and merge it with output from `getSpiFlashAddresses`. (not in this PR though)